library/src/main/java/com/schibsted/spain/barista/custom/PerformClickAction.java

```diff
@@ -32,7 +33,11 @@ public String getDescription() {
 
       @Override
       public void perform(UiController uiController, View view) {
-        view.performClick();
+        if (view.isClickable()) {
+          view.performClick();
+        } else {
+          click().perform(uiController, view);
+        }
```

Comment:
As this is a bit of magic, we should explain why are we doing this trick. I don't have any suggestions but I feel that project newcomers will cry when they see this thing.